internal/oci/runtime_oci.go

```diff
@@ -892,6 +893,17 @@ func (r *runtimeOCI) UpdateContainerStatus(c *Container) error {
 	oomFilePath := filepath.Join(c.bundlePath, "oom")
 	if _, err = os.Stat(oomFilePath); err == nil {
 		c.state.OOMKilled = true
+
+		// Collect total metric
+		metrics.CRIOContainersOOMTotal.Inc()
+
+		// Collect metric by container name
+		counter, err := metrics.CRIOContainersOOM.GetMetricWithLabelValues(c.Name())
+		if err != nil {
+			log.Warnf(ctx, "Unable to write OOM metric by container: %v", err)
```

Comment:
we haven't wired ctx throug hyet, so this should be `logrus.Warnf("Unable...")`

Reply:
Fixed